datadog_checks_base/tests/base/checks/openmetrics/test_v2/utils.py

```diff
@@ -0,0 +1,21 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import OpenMetricsBaseCheckV2
+
+
+def get_check(instance=None, init_config=None):
+    return _get_check(OpenMetricsBaseCheckV2, instance, init_config)
+
+
+def _get_check(cls, instance, init_config):
```

Comment:
Not a big deal, but since this function is now called only once (legacy tests use their own version), would it make sense to merge it with `get_check`?